### PR TITLE
Quiz2 Submit

### PR DESCRIPTION
**Pull request by arrijabba**
Arri Manuel, aem88

### CODE REVIEW

**Comment by matty-digital** [comment]
easy on those commit messages...changes are always just a `git pull` away

File: template/quiz2.html [hunk 1]
```diff
@@ -3,35 +3,56 @@
 	<head>
 		<meta charset="UTF-8"
```
Comment (matty-digital) [inline template/quiz2.html]: missing closing `>`

File: template/quiz2.html [hunk 2]
```diff
@@ -3,35 +3,56 @@
...
 		<title>Quiz 2</title>
+
+		<link href="quiz2.css" rel="stylesheet" type="text/css"/>
+
 	</head>
 	<body>
 		
 		<div class="content">
```
Comment (matty-digital) [inline template/quiz2.html]: missing closing tag

File: template/quiz2.js [hunk 3]
```diff
@@ -1,37 +1,89 @@
...
+	$button3   = $('.quizBtn3');
+	var random;
+
 
 	$mouseover.on('mouseover', function() {
 		$this = $(this);
-		$(this).html('Scrooge McDuck!');
-		$(this).height($(this).height() + 50);
+		$(this).text('Scrooge McDuck!');
+		//$(this).height($(this).height() + 50);
```
Comment (matty-digital) [inline template/quiz2.js]: try not to include commented code and/or debug statements

File: template/quiz2.js [hunk 4]
```diff
@@ -1,37 +1,89 @@
...
+
+	$('.createButton').on('click', quizFunction);
+
+
+	$('.quizBtn2').on('click', function(){
+		document.cookie = 'quiz=' + random;
+	});
+
+
+	cookie_val = document.cookie.split("=");
```
Comment (matty-digital) [inline template/quiz2.js]: var

File: template/quiz2.js [hunk 5]
```diff
@@ -1,37 +1,89 @@
...
+	if(parseInt(cookie_val[1]) >= 0)
+	{
+		//quizFunction();
```
Comment (matty-digital) [inline template/quiz2.js]: comments

File: template/quiz2.js [hunk 6]
```diff
@@ -1,37 +1,89 @@
...
+			.done(function( data ) {
+
+				$('.retData').append(data.data[cookie_val[1]]);
+
+ 			 });
+	}
+
+
+	$(document).ready(function() {
+		$button2.hide();
```
Comment (matty-digital) [inline template/quiz2.js]: the IIFE this file is wrapped in happens when the document is ready

File: template/quiz2.html (hunk 2 above)
Comment (arrijabba) [inline template/quiz2.html]: Link tag is self closing, unless you're talking about something different

File: template/quiz2.js (hunk 4 above)
Comment (arrijabba) [inline template/quiz2.js]: variable init without var makes it a global. which still works here.

File: template/quiz2.js (hunk 5 above)
Comment (arrijabba) [inline template/quiz2.js]: doesn't say I couldn't include comments in instructions

File: template/quiz2.js (hunk 3 above)
Comment (arrijabba) [inline template/quiz2.js]: doesn't say I couldn't include comments in instructions

File: template/quiz2.html (hunk 2 above)
Comment (matty-digital) [inline template/quiz2.html]: I was, this was in reference to `<div class="content">`. I'm aware of html spec for self closing tags.
The spacing was a little tough to read as well.

File: template/quiz2.js (hunk 3 above)
Comment (matty-digital) [inline template/quiz2.js]: This follows multiple in-class discussions on best practices

File: template/quiz2.js (hunk 4 above)
Comment (matty-digital) [inline template/quiz2.js]: I'm familiar with variable declarations and what makes them global or local. I also didn't say it wouldn't work. I also mentioned in class that when you see comments on the pull requests, that some of them were just comments

File: template/quiz2.js (hunk 5 above)
Comment (matty-digital) [inline template/quiz2.js]: see above comments, this was covered in class repeatedly

**Pull request closed without merging**